archiver.go

```diff
@@ -624,35 +609,23 @@ func writeRunRecords(ctx context.Context, db *sqlx.DB, archive *Archive, writer
 	var rows *sqlx.Rows
 	rows, err := db.QueryxContext(ctx, lookupFlowRuns, archive.Org.IsAnon, archive.Org.ID, archive.StartDate, archive.endDate())
 	if err != nil {
-		return 0, err
+		return 0, errors.Wrapf(err, "error querying run records for org: %d", archive.Org.ID)
 	}
 	defer rows.Close()
 
 	recordCount := 0
-	var record, visibility string
+	var record string
 	var exitedOn *time.Time
 	for rows.Next() {
-		if archive.ArchiveType == MessageType {
```

Comment:
MessageType block was never entered, when we added broadcast deleting we changed to our own method.